test/unit/uk/gov/hmrc/exports/controllers/request/ExportsDeclarationRequestSpec.scala

```diff
@@ -81,13 +81,13 @@ class ExportsDeclarationRequestSpec extends UnitSpec with ExportsDeclarationBuil
     natureOfTransaction = Some(natureOfTransaction)
   )
 
-  "Request" should {
+  "ExportsDeclarationRequest" should {
     "map to ExportsDeclaration" in {
-      request.toExportsDeclaration(id, Eori(eori)) mustBe declaration
+      ExportsDeclaration(id, Eori(eori), exportsDeclarationRequest) mustBe exportsDeclaration
```

Comment:
This spec is for `ExportsDeclarationRequest`, but you are testing `ExportsDeclaration` here.